localization/yabloc/yabloc_pose_initializer/src/semantic_segmentation/semantic_segmentation_server.py

```diff
@@ -23,6 +24,12 @@
 from sensor_msgs.msg import Image
 from yabloc_pose_initializer.srv import SemanticSegmentation
 
+# cspell: ignore DDOWNLOAD
+error_message = """\
```

Comment:
Nits: better to make it capital?
```suggestion
ERROR_MESSAGE = """\
```

Reply:
OK :ok_hand:  I have already fixed it in c809a20